packages/shopify-codemod/test/transforms/function-to-arrow.test.js

```diff
@@ -0,0 +1,12 @@
+import 'test-helper';
+import ternaryStatementToIfStatement from 'function-to-arrow';
+
+describe('functionToArrow', () => {
+  it('transforms the simple case', () => {
+    expect(ternaryStatementToIfStatement).to.transform('function-to-arrow/basic');
+  });
+
+  it("doesn't transform function that use `this`", () => {
+    expect(ternaryStatementToIfStatement).to.transform('function-to-arrow/this');
+  });
```

Comment:
but I don't think we should support that specific case, probably doesn't happen too often and it's more trouble than it's worth